book/src/snapshot-restore-feature.md

```diff
@@ -146,6 +146,10 @@ Create a _PVC_ from a _VolumeSnapshot_:
 kuberctl create -f restore.yaml
 ```
 
+To create an invalid VolumeSnapshot:
```

Comment:
We don't want to encourage user to create an invalid volume snapshot.  This is an example to test the validation webhook feature.  I don't think we need to add it here.  It will actually add confusion.

Reply:
@xing-yang sure , my intention was here to brief user about the example yaml for an invalid  volume snapshot , i am totally agreed with you if it is creating confusion, then this change should not be implemented here

Reply:
/close